Workshop

creighton opened the pull request.

creighton committed:
Merge pull request #1 from adlnet/master

update my fork

creighton committed:
updates for workshop and launch

creighton committed:
fixing stuff

creighton committed:
fixing stuff

creighton committed:
fixing stuff

creighton committed:
added packaged folder

ljwolford reviewed (commented):
Should the `http://adlnet.gov/event/2015/xapibootcamp` URI be updated to at least 2016? Is this the URI to show before showing how to use launch?

Is there a way to highlight the one-line adds in the code examples?

I thought I saw a document.ready bit in there that isn't needed.

creighton commented:
are you looking at the reverse side? i removed all the baseuris.. it'll be given to the content from the launch server. and the document ready thing was removed when i moved the code to the end of the html

ljwolford commented:
I could have sworn I was looking at your branch but now I see I wasn't....will try again

creighton committed:
Update Readme.md

creighton committed:
Update Readme.md

creighton committed:
fixed stuff in final
